go.mod

```diff
@@ -5,5 +5,6 @@ go 1.19
 require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc4
+	github.com/oras-project/oras-credentials-go v0.3.0
```

Comment:
I'm worried about this import as it may cause import cycle

Reply:
removed this example, and now no changes to go.mod.